include/mxnet/tuple.h

```diff
@@ -653,6 +661,13 @@ inline bool shape_is_known(const TShape& x) {
   return true;
 }
 
+inline bool shape_is_known(const std::vector<TShape>& shapes) {
```

Comment:
why inline?

Reply:
Removing `inline` here actually would result in a linker error.

Reply:
I see there's no tuple implementation file. I guess it's not worth it to add it for a small function.